benchmark/sqlite/sqlite-prepare-select-all.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
+const common = require('../common.js');
+const sqlite = require('node:sqlite');
+const assert = require('assert');
+
+const bench = common.createBenchmark(main, {
+  n: [1e5],
+  statement: [
+    'SELECT 1',
+    'SELECT * FROM foo LIMIT 1',
+    'SELECT * FROM foo LIMIT 100',
+    'SELECT text_column FROM foo LIMIT 1',
+    'SELECT text_column FROM foo LIMIT 100',
+    'SELECT text_column, integer_column FROM foo LIMIT 1',
+    'SELECT text_column, integer_column FROM foo LIMIT 100',
+    'SELECT text_column, integer_column, real_column FROM foo LIMIT 1',
+    'SELECT text_column, integer_column, real_column FROM foo LIMIT 100',
+    'SELECT text_column, integer_column, real_column, blob_column FROM foo LIMIT 1',
+    'SELECT text_column, integer_column, real_column, blob_column FROM foo LIMIT 100',
+    'SELECT text_8kb_column FROM foo_large LIMIT 1',
+    'SELECT text_8kb_column FROM foo_large LIMIT 100',
+  ],
+});
+
+function main(conf) {
+  const n = conf.n | 0;
+
+  const db = new sqlite.DatabaseSync(':memory:');
+
+  db.exec('CREATE TABLE foo (text_column TEXT, integer_column INTEGER, real_column REAL, blob_column BLOB)');
+  const fooInsertStatement = db.prepare(
+    'INSERT INTO foo (text_column, integer_column, real_column, blob_column) VALUES (?, ?, ?, ?)',
+  );
+
+  for (let i = 0; i < 2e6; i++) {
```

Comment:
Why do we have a magical number here?

Reply:
This is used just to populate the table, no particular reason for any number, it just needs to be greather than what we are selecting.

Initially, I added tests with limit 10000, but it was too slow to run, so I reduced the amount of runs in the benchmark to 1e5 and reduced the limit to 100.

I can move this to the conf but I don't think it make sense.

Reply:
Nevermind, changing to conf as `tableSeedSize`